pkg/bootstrap/bootstrap_config_test.go

```diff
@@ -447,12 +448,8 @@ func loadProxyConfig(base, out string, _ *testing.T) (*meshconfig.ProxyConfig, e
 	}
 
 	// Exported from makefile or env
-	cfg.ConfigPath = out + "/bootstrap/" + base
-	gobase := os.Getenv("ISTIO_GO")
-	if gobase == "" {
-		gobase = "../.."
-	}
-	cfg.CustomConfigFile = gobase + "/tools/packaging/common/envoy_bootstrap_v2.json"
+	cfg.ConfigPath = filepath.Join(out, "bootstrap", base)
+	cfg.CustomConfigFile = filepath.Join(env.IstioSrc, "tools", "packaging", "common", "envoy_bootstrap_v2.json")
```

Comment:
Same - please allow the use of a work directory, no more temp/test/etc files in the source tree.

Reply:
I don't think this change does what you're afraid of? It used to compute "almost" GOPATH from ISTIO_GO and then use it to find Istio sources. This does exactly that, using the right value from env package.